### DISCUSSION TITLE
DataGrid Data soures with Nestede List

### DISCUSSION THREAD

**Discussion by peterhym21** [Megabit/Blazorise#3857] [Q&A] [answered]
Hey Is it possible to show data for a nested List inside the DataGrid in a column 
all the data is being provided for the DB and it is in the "Operations" list which is the DataGrid Data

But when I run the project it just shows the "UiOperationListVm.Description" insted of "UiOperationListVm.OperationType.Name"

                            <DataGrid TItem="UiOperationListVm"
                                  Data="@Operations"
                                  @bind-SelectedRow="@SelectedOperation"
                                  EditMode="DataGridEditMode.Inline"
                                  Editable
                                  PageSize="10"
                                  ShowPager
                                  Responsive>

                                <DataGridColumn Field="@nameof(UiOperationListVm.Id)" Caption="ID" Sortable="false" Displayable="true" />
                                <DataGridColumn Field="@nameof(UiOperationListVm.Name)" Caption="Name" Editable />
                                <DataGridColumn Field="@nameof(UiOperationListVm.Description)" Caption="Description" Editable />
                                // this is the nested List I want shown
                                <DataGridColumn Field="@nameof(UiOperationListVm.OperationType.Name)" Caption="Operation types" Editable>
                                    <EditTemplate>
                                        <SelectList TItem="UiOperationTypeListVm"
                                                TValue="int"
                                                Data="@OperationTypeListVm"
                                                TextField="@((item)=>item.Description)"
                                                ValueField="@((item)=>(int)item.Name)"
                                                SelectedValue="@UpdateOperationCommand.OperationTypeName"
                                                SelectedValueChanged="@MyListValueChangedHandler"
                                                DefaultItemText="test" />
                                    </EditTemplate>
                                </DataGridColumn>
                                <DataGridCommandColumn>
                                    <EditCommandTemplate>
                                        <Button Color="Color.Primary" Clicked="@context.Clicked">Edit</Button>
                                    </EditCommandTemplate>
                                    <DeleteCommandTemplate>
                                        <Button Color="Color.Primary" Clicked="@(()=>OnDeletClickedOperation(context))">@context.LocalizationString</Button>
                                    </DeleteCommandTemplate>
                                    <SaveCommandTemplate>
                                        <Button Color="Color.Primary" Clicked="@(()=>OnSaveClickedOperation(context))">@context.LocalizationString</Button>
                                    </SaveCommandTemplate>
                                    <CancelCommandTemplate><Button Color="Color.Danger" Clicked="@context.Clicked">Cancel</Button></CancelCommandTemplate>
                                    <NewCommandTemplate></NewCommandTemplate>

                                </DataGridCommandColumn>
                            </DataGrid>

**Comment by stsrki** [comment, marked as answer]
Yes it is possible. You need to define it without `nameof`

Example

```
<DataGridColumn Field="OperationType.Name" />
```
